opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -1066,6 +1066,18 @@ type SettingsMessage {
   recipients: [Member!] @auth(for: [SETTINGS])
 }
 
+enum PlatformCriticalAlertType {
+  GROUP_WITH_NULL_EFFECTIVE_LEVEL
+}
+type PlatformCriticalAlertDetails {
+  groups: [Group!]!
+}
```

Comment:
this is where the critical alert system is a bit limited, not well generalized. details shall depends on type.

I did not have time to overengineer this system, honestly it's a quick bandaid to help admins prepare for 6.0.

Reply:
Not a problem, thats purely dynamic and so easy to change.